core-web/libs/portlets/dot-analytics/portlet/project.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "portlets-dot-analytics",
+    "name": "analytics-portlets",
```

Comment:
Other portlets follow the prefix `portlets-[my-portlet]`:

<img width="434" height="189" alt="Image" src="https://github.com/user-attachments/assets/1f1f49c1-f216-4b8f-a0d0-227673335c56" />